src/test/java/bio/terra/common/fixtures/DatasetFixtures.java

```diff
@@ -100,4 +104,32 @@ public static DatasetRequestModel buildDatasetRequest() {
             .defaultProfileId(UUID.randomUUID().toString())
             .schema(buildSchema());
     }
+
+    /**
+     * Use method to generate a DatasetTable.  Individual values can be mutated by tests if needed
+     */
+    public static DatasetTable generateDatasetTable(String name,
+                                                    TableDataType baseType,
+                                                    List<String> columnNames) {
+        if (columnNames.isEmpty()) {
```

Comment:
In test code, I would use `assertThat()` rather than `if (...) { throw new ... }`. There's no need to create an exception and throw it since nobody will catch it.

Although, what is the problem with creating a dataset table with no columns? Does that cause problems in other parts of our code?

Reply:
totally.  You'd end up creating tables in bigquery with no columns and that will totally fail

Reply:
OK, so then the test would fail at that point. No need to check it here if it eventually fails.